Analysis/Core/include/AnalysisCore/RecoDecay.h

```diff
@@ -874,7 +874,7 @@ class RecoDecay
         *listIndexDaughters = arrAllDaughtersIndex;
       }
     }
-    //Printf("MC Gen: Accepted: m: %d", candidate.globalIndex());
+    //Printf("MC Gen: Accepted: m: %ld", candidate.globalIndex());
```

Comment:
Does this one really trigger a warning? Index should be `int`.

Reply:
I don't recall, I changed it back to int. One will see when (s)he uncomments the line, it might be that I changed it by mistake.